src/core/objects/object.cpp

```diff
@@ -64,6 +64,27 @@ namespace literal
 
 
 
+namespace OpenOrienteering {
+
+namespace {
+	
+/** 
+ * Checks if a point is on the segment defined by the given start and end coordinates.
+ */
+bool isPointOnSegment(const MapCoordF& seg_start, const MapCoordF& seg_end, const MapCoordF& point)
```

Comment:
All unnamed namespaces are used intentionally, to avoid external linkage when it is not  neccessary. In particular, this function was moved away from a location were it was not used, and the anonymous namespace helped to reconfirm that there is no other usage.

Reply:
Understood. My point was that I'd like to see a brief note about the move in commit message so that it's clear that the move was indeed intentional. Again, this serves as a hint for future contributors.